src/backend/utils/parse-router.ts

```diff
@@ -50,5 +50,11 @@ function assignChildrenToParent(parent, children): [any] {
 }
 
 function childRouteName(child): string {
-  return child.component ? child.component.name : 'no-name-route';
+  if (child.component) {
```

Comment:
@igor-ka My thinking was that because they are returns, it would be a little redundant, but that's not a strong opinion. Change made.